home/templates/home/tags/top_level_sections.html

```diff
@@ -0,0 +1,11 @@
+{% load static wagtailcore_tags wagtailimages_tags i18n %}
+<footer class="top-level-sections">
+    <nav>
+        <div class="links">
+          {% for section in top_level_sections %}
+            {% image section.specific.lead_image fill-20x20 %}
```

Comment:
On the menu we display the icon not the lead image
```suggestion
            {% image section.specific.icon fill-20x20 %}
```